spec/models/provider_spec.rb

```diff
@@ -61,4 +61,32 @@
       expect(tenant.providers).to include(provider)
     end
   end
+
+  context "#destroy_queue" do
+    before do
+      allow(Time).to receive(:now).and_return(Time.zone.now)
```

Comment:
Is this necessary?

Reply:
yes, so that the `15.seconds.from_now` check would be matched.

Reply:
Using the `MiqQueue.where(…)`.to have_attributes would solve this.

Reply:
ok, just did what you've requested